sessions/backend-create/demo-end/db/connect.js

```diff
@@ -0,0 +1,47 @@
+import mongoose from "mongoose";
```

Comment:
Hey there :) I've investigated a little bit, and it seems that your suggestion uses MongoClient instead of mongoose? I tried to research whether or not the two are interchangeable, but it appears that they work differently, while mongoose is using MongoClient "under the hood". Since we've used mongoose so far, I'd be a little hesitant to change it? The [db/connent.js ](https://github.com/vercel/next.js/blob/canary/examples/with-mongodb-mongoose/lib/dbConnect.js)here utilizing mongoose also seems to be the one that we're using currently. I'd appreciate your insight. :)

Reply:
true true, my bad. please ignore